avm/utilities/pipelines/platform/Assign-Issue.ps1

```diff
@@ -0,0 +1,111 @@
+function Get-AvmCsv {
+  
+  [CmdletBinding()]
+  param (
+    [Parameter(Mandatory)]
+    [ValidateSet('Bicep-Resource', 'Bicep-Pattern', 'Terraform-Resource', 'Terraform-Pattern')]
+    [string] $ModuleIndex
+  )
+
+  # Retrieve the CSV file
+  if ($ModuleIndex -eq 'Bicep-Resource') {
+    try {
+      $unfilteredCSV = Invoke-WebRequest -Uri "https://aka.ms/avm/index/bicep/res/csv"
+    }
+    catch {
+      Write-Error "Unable to retrieve CSV file - Check network connection."
+    }
+  }
+  elseif ($ModuleIndex -eq 'Bicep-Pattern') {
+    try {
+      $unfilteredCSV = Invoke-WebRequest -Uri "https://aka.ms/avm/index/bicep/ptn/csv"
+    }
+    catch {
+      Write-Error "Unable to retrieve CSV file - Check network connection."
+    }
+  }
+  elseif ($ModuleIndex -eq 'Terraform-Resource') {
+    try {
```

Comment:
Would the terraform counterpart ever be used here? If not needed I'd suggest removing those lines for cleaner code

Reply:
Done